Update turn-off-microsoft-usage-data-collection-windows.md (#3649)
MicrosoftDocs/memdocs: memdocs/intune/user-help/turn-off-microsoft-usage-data-collection-windows.md

When set to **Yes**, your in-app performance and usage data is automatically collected and shared with Microsoft. This type of data helps us improve the reliability and performance of Microsoft products. If you disable the usage data collection, your organization cannot change your preferences.    

> For more information on what data is being collected, check [Optional diagnostic data from Intune Client apps](/mem/intune/protect/client-apps-optional-data)

Reviewer: ```suggestion
For more information about the data that we collect, see [Optional diagnostic data from Intune Client apps](/mem/intune/protect/client-apps-optional-data).
```
